src/components/NFTPortfolio/SubscriptionForm.vue

```diff
@@ -166,6 +167,25 @@ export default {
       }
       return 'portfolio_follow_hint';
     },
+    unverifiedEmailLink() {
```

Comment:
How about this, `unverifiedEmailLink` seems ambiguous to me
```suggestion
    checkEmailLink() {
```